ppr-ui/src/components/mhrRegistration/YourHome/ManufacturerMakeModel.vue

```diff
@@ -219,6 +227,13 @@ export default defineComponent({
       )
     )
 
+    const manufacturerNameRules = computed((): Array<Function> =>
+      customRules(
+        required("Enter a manufacturer's name or enter \"Unknown\" "),
```

Comment:
Design calls for this error message? 
`Enter a manufacturer's name or enter "Unknown"`

Reply:
Sounds good, thanks for verifying!